resources/js/views/settings/users/View.vue

```diff
@@ -116,6 +116,25 @@
                 </b-form-select>
                 <template slot='invalid-feedback'><div v-html="fieldError('user_locale')"></div></template>
               </b-form-group>
+              <b-form-group
+                label-cols-sm='3'
+                :label="$t('settings.users.timezone')"
+                label-for='timezone'
+                :state='fieldState("timezone")'
+              >
+                <b-form-select
+                  :options='model.timezones'
```

Comment:
This doesn't work for new users as the timezone list is never loaded. I'm already working on a global default timezone setting that can pre-select this timezone field.